src/database/schema/ChatSchema.ts

```diff
@@ -30,6 +30,8 @@ class ChatSchema implements BaseDbSchema {
 
   isContinuous: boolean;
 
+  attachmentJson: any;
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachmentJson` property is declared as `any`. It's better to avoid using `any` type in TypeScript because it bypasses type checking. If the structure of the attachments is known, you should create an interface for it.
```diff
-  attachmentJson: any;
+  attachmentJson: AttachmentInterface; // Define this interface according to your needs
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->